coala_quickstart/info_extractors/GruntfileInfoExtractor.py

```diff
@@ -0,0 +1,292 @@
+import logging
+
+from pyjsparser import PyJsParser
+
+from coala_quickstart.info_extraction.InfoExtractor import InfoExtractor
+from coala_quickstart.info_extraction.Information import (
+    IncludePathsInfo, IgnorePathsInfo, LintTaskInfo, MentionedTasksInfo)
+from coala_quickstart.info_extractors.Utilities import (
+    search_object_recursively)
+
+
+class GruntfileInfoExtractor(InfoExtractor):
+    supported_file_globs = ("Gruntfile.js",)
+    supported_info_kinds = (LintTaskInfo, MentionedTasksInfo)
+
+    def parse_file(self, fname, file_content):
+        js_parser = PyJsParser()
+        return js_parser.parse(file_content)
+
+    def find_information(self, fname, parsed_file):
+        results = []
+
+        npm_tasks = self.get_npm_tasks(parsed_file)
+        linters = self.extract_lint_subtasks(parsed_file)
+        config = self.get_configurations(parsed_file, linters)
+
+        if npm_tasks:
+            results.append(
+                MentionedTasksInfo(fname, npm_tasks))
+
+        for linter in linters:
+            include_paths_info = None
+            ignore_paths_info = None
+            if config.get(linter):
+                linter_config = config[linter]
+                globs = self.extract_globs(linter_config)
+                linter_config = self.get_task_config(linter_config)
+                if globs.get("include_globs"):
+                    include_paths = globs["include_globs"]
+                    include_paths_info = IncludePathsInfo(
+                        fname, include_paths) if include_paths else None
+                if globs.get("ignore_globs"):
+                    ignore_paths = globs["ignore_globs"]
+                    ignore_paths_info = IgnorePathsInfo(
+                        fname, ignore_paths) if ignore_paths else None
+            results.append(
+                LintTaskInfo(fname,
+                             linter,
+                             None,
+                             include_paths_info,
+                             ignore_paths_info,
+                             linter_config))
```

Comment:
`linter_config` could be undefined over here. Do a `linter_config = None` before the `if`.